lib/mutest/ast/meta/send.rb

```diff
@@ -65,6 +66,10 @@ def receiver_possible_top_level_const?
           Const.new(receiver).possible_top_level?
         end
 
+        def method_selector?
```

Comment:
What about something like `method_object_selector?` Might be a bit more clear. If you don't like that then it might be worth adding a small clarifying comment saying it means it is looking for the method that returns a `Method` object